In [1]:
import pylab as plt
import math
x1=[]
x2=[]
x3=[]
x4=[]

In [2]:
def priEcua():
    x=-5
    y=0    
    while (x<=15):   
        print x, y
        y= 9.5 + ( 0.5*x )
        x=x+0.5
        x1.append(x)
        x2.append(y)        
        
def segEcua():
    xx=-5
    yy=0
    while (xx<=15): 
        print xx, yy
        yy= 9.4 + ( xx*0.51 )
        xx=xx+0.5
        x3.append(xx)
        x4.append(yy)
        
print "primera ecuacion"
priEcua()
print "segunda ecuacion"
segEcua()

primera ecuacion
-5 0
-4.5 7.0
-4.0 7.25
-3.5 7.5
-3.0 7.75
-2.5 8.0
-2.0 8.25
-1.5 8.5
-1.0 8.75
-0.5 9.0
0.0 9.25
0.5 9.5
1.0 9.75
1.5 10.0
2.0 10.25
2.5 10.5
3.0 10.75
3.5 11.0
4.0 11.25
4.5 11.5
5.0 11.75
5.5 12.0
6.0 12.25
6.5 12.5
7.0 12.75
7.5 13.0
8.0 13.25
8.5 13.5
9.0 13.75
9.5 14.0
10.0 14.25
10.5 14.5
11.0 14.75
11.5 15.0
12.0 15.25
12.5 15.5
13.0 15.75
13.5 16.0
14.0 16.25
14.5 16.5
15.0 16.75
segunda ecuacion
-5 0
-4.5 6.85
-4.0 7.105
-3.5 7.36
-3.0 7.615
-2.5 7.87
-2.0 8.125
-1.5 8.38
-1.0 8.635
-0.5 8.89
0.0 9.145
0.5 9.4
1.0 9.655
1.5 9.91
2.0 10.165
2.5 10.42
3.0 10.675
3.5 10.93
4.0 11.185
4.5 11.44
5.0 11.695
5.5 11.95
6.0 12.205
6.5 12.46
7.0 12.715
7.5 12.97
8.0 13.225
8.5 13.48
9.0 13.735
9.5 13.99
10.0 14.245
10.5 14.5
11.0 14.755
11.5 15.01
12.0 15.265
12.5 15.52
13.0 15.775
13.5 16.03
14.0 16.285
14.5 16.54
15.0 16.795


In [3]:
uno = plt.plot(x1,x2,"b", label="primera ecuacion")
dos = plt.plot(x3,x4,"y", label="segunda ecuacion")
plt.title("Funcion")   
plt.xlabel("Valores en x")
plt.ylabel("Valores en y")
plt.grid(True)  
plt.legend()
plt.show()

###b) 
$D = 
\begin{vmatrix}
0.5 & -1\\
1.02 & -2
\end{vmatrix}
$
=
$(0.5)*(-2)-(1.02)*(-1) = 0.02$
###c) $ \text{Como las pendientes de ambas ecuaciones son muy cercanas es complicado observar el punto de intersección. Por el determinante se puede concluir que las ecuaciones son linealmente independientes.}$
###d)
$$
x_1 = \frac{(-2)(-9.5)-(-1)(-18.8)}{(0.5)(-2)-(-1)(1.02)} = 10 $$
$$
x_2 = \frac{(0.5)(-18.8)-(1.02)(-9.5)}{(0.5)(-2)-(-1)(1.02)} = 14.5
$$
###e)
$$
x_1 = \frac{(-2)(-9.5)-(-1)(-18.8)}{(0.52)(-2)-(-1)(1.02)} = -10 $$
$$
x_2 = \frac{(0.52)(-18.8)-(1.02)(-9.5)}{(0.52)(-2)-(-1)(1.02)} = 4.3
$$
$$ \text{El valor de las incógnitas cambia significativamente, dependiendo de las cifras decimales de los coeficientes, los valores que satisfacen la ecuación no afectaran drásticamente el comportamiento de la función}$$